types/cache.interfaces.ts

```diff
@@ -26,3 +26,6 @@ export type GetResult = {
     hit: boolean;
     data?: any;
 };
+
+// Something that can be used as a value and passed to cache functions. A simple JSON-able type.
+export type CacheableValue = string | number | boolean | {[x: string]: CacheableValue} | Array<CacheableValue>;
```

Comment:
More accurate type is `{[x: string | number]: CacheableValue}` for the object type

Reply:
I'm happy not to handle this case :D